distributed/diagnostics.py

```diff
@@ -449,15 +454,24 @@ def clear_errors(errors):
             errors = sync(self.scheduler.loop, self.setup, keys, complete)
 
         # Set up widgets
-        from ipywidgets import FloatProgress, VBox, HTML, HBox
-        self.bars = {key: FloatProgress(min=0, max=1, description=key)
-                        for key in self.all_keys}
-        self.texts = {key: HTML() for key in self.all_keys}
-        self.boxes = {key: HBox([self.bars[key], self.texts[key]])
+        from ipywidgets import FloatProgress, HBox, VBox, HTML
+        self.elapsed_time = HTML('')
+        self.bars = {key: FloatProgress(min=0, max=1, description='', height = '10px')
                         for key in self.all_keys}
-        self.time = HTML()
-        self.widget = HBox([self.time, VBox([self.boxes[key] for key in
-                                            sorted(self.bars, key=str)])])
+        self.bar_texts = {key: HTML('') for key in self.all_keys}
+        self.bar_labels = {key: HTML('<div style=\"padding: 0px 10px 0px 10px; text-align:right; word-wrap: break-word;\">' + key + ':</div>', width = '300px') for key in self.all_keys}
```

Comment:
We should probably escape `key` here.  I noticed that `<lambda>` wasn't printing correctly (though I should probably create better names for lambdas.)